library/std/src/sys/os_str/utf8.rs

```diff
@@ -0,0 +1,329 @@
+//! The underlying OsString/OsStr implementation on Motor OS is UTF-8.
```

Comment:
Mind updating this to just say it's a guaranteed UTF-8 OsStr, rather than mentioning Motor? Since there's a good chance we'll use it for other platforms in the future.